toc.json

```diff
@@ -1,83 +1,9 @@
 {
   "items": [
-    {
-      "type": "item",
-      "title": "Quickstart",
-      "uri": "docs/1-Quickstart.md"
-    },
-    {
-      "type": "divider",
-      "title": "Integrating with Finch"
-    },
-    {
-      "type": "item",
-      "title": "How Finch Works",
-      "uri": "docs/Integrating-with-Finch/How Finch Works.md"
-    },
-    {
-      "type": "group",
-      "title": "Integrate Finch Connect",
-      "items": [
-        {
-          "type": "item",
-          "title": "Overview",
-          "uri": "docs/Integrating-with-Finch/Integrate-Finch-Connect/Overview.md"
-        },
-        {
-          "type": "item",
-          "title": "Redirecting to Connect",
-          "uri": "docs/Integrating-with-Finch/Integrate-Finch-Connect/Redirect-to-Connect.md"
-        },
-        {
-          "type": "item",
-          "title": "Embedding Connect",
-          "uri": "docs/Integrating-with-Finch/Integrate-Finch-Connect/Embed-Connect.md"
-        }
-      ]
-    },
-    {
-      "type": "item",
-      "title": "Integrate the Finch API",
-      "uri": "docs/Integrating-with-Finch/Integrate-the-Finch-API.md"
-    },
-    {
-      "type": "item",
-      "title": "Integration Checklist",
-      "uri": "docs/Integrating-with-Finch/Integration Checklist.md"
-    },
-    {
-      "type": "divider",
-      "title": "Product Guides"
-    },
-    {
-      "type": "item",
-      "title": "Automated Connect Flow",
-      "uri": "docs/Product-Guides/Automated-Connect-Flow.md"
-    },
-    {
-      "type": "item",
-      "title": "Assisted Connect Flow",
-      "uri": "docs/Product-Guides/Assisted-Connect-Flow.md"
-    },
-    {
-      "type": "item",
-      "title": "Benefits API",
-      "uri": "docs/Product-Guides/Benefits-API.md"
-    },
-    {
-      "type": "item",
-      "title": "Request Forwarding",
-      "uri": "docs/Product-Guides/Request-Forwarding.md"
-    },
     {
       "type": "divider",
       "title": "Development Guides"
     },
-    {
-      "type": "item",
-      "title": "Providers",
-      "uri": "docs/Development-Guides/Providers.md"
-    },
     {
       "type": "item",
       "title": "Redirect URIs",
```

Comment:
Redirect URIs can be removed as well. They were merged into the Create Account doc in the Implementation Guide, and I don't believe they really belong in the API Reference since it is more of a Finch Connect concept.